config.rb

```diff
@@ -114,9 +120,9 @@ def deploy_to target
 require 'rspec_info/helpers'
 helpers RSpecInfo::Helpers
 
-RSpecInfo::Helpers.rspec_documentation_latest(source_dir).each do |gem_name, version|
-  Dir.glob(File.join(source_dir, "/documentation/#{version}/#{gem_name}/**/*")).select { |f| File.file? f }.each do |f|
-    relative_path = Pathname.new(f).relative_path_from(Pathname.new(source_dir))
+RSpecInfo::Helpers.rspec_documentation_latest(app.source_dir).each do |gem_name, version|
+  Dir.glob(File.join(app.source_dir, "/documentation/#{version}/#{gem_name}/**/*")).select { |f| File.file? f }.each do |f|
```

Comment:
I ended up with `**/*.html`, otherwise it was replacing CSS and JS with some nonsense.

Reply:
I mean this is supposed to replace css and js...